install/operator/pkg/cmd/internal/grant/grant.go

```diff
@@ -49,6 +51,7 @@ func New(parent *internal.Options) *cobra.Command {
 
 	cmd.PersistentFlags().BoolVarP(&o.cluster, "cluster", "", false, "add the permission for all projects in the cluster(requires cluster admin privileges)")
 	cmd.PersistentFlags().StringVarP(&o.User, "user", "u", pkg.DefaultOperatorImage, "add permissions for the given User")
+	cmd.PersistentFlags().BoolVarP(&o.publicApi, "publicApi", "", false, "add permissions required for publicApi addon in the cluster(requires cluster admin privileges)")
```

Comment:
No need for this, look at following comment.

Reply:
If the ClusterRoleBinding is moved to the addon, the operator's service-account has to hold the same permissions in it's own list of roles. I tried that, and it's actually worse, since it turns into a laundry list of other permissions the operator must hold. 
Not to mention that if we do it this way, only a cluster admin could install the Syndesis operator preventing developers from being able to setup private development environments in their own namespaces. 
I think it's better to separate concerns and keep this cluster admin task out of the operator's runtime and to instead make it something it can do externally on behalf of a cluster admin through the grant command.